src/engine/outfit.ts

```diff
@@ -147,12 +157,29 @@ export function avoidDaylightShavingsHelm(): boolean {
   );
 }
 
+// eslint-disable-next-line libram/verify-constants
+const candySword = $item`candy cane sword cane`;
+
+function useCandyCaneSword(): boolean {
+  if (!have(candySword)) return false;
+  if (get("instant_saveCandySword", false)) return false;
+  examine(candySword);
+  // eslint-disable-next-line libram/verify-constants
+  if (haveEffect($effect`Peppermint Rush`) >= 401) return false;
+  if (get("_surprisinglySweetSlashUsed", 0) < 11 || get("_surprisinglySweetStabUsed", 0) < 11) {
+    return true;
+  }
+  return false;
+}
+
 export function baseOutfit(allowAttackingFamiliars = true): OutfitSpec {
   // Only try equipping/nag LOV Epaulettes if we are done with the LOV tunnel
   const lovTunnelCompleted = get("_loveTunnelUsed") || !get("loveTunnelAvailable");
 
   return {
-    weapon: have($item`fish hatchet`)
+    weapon: useCandyCaneSword()
```

Comment:
I'm not seeing how the candy cane skills are being used when adding this to `baseOutfit` (aside from the special handling that was already added for snojo and rift)

Reply:
Fixed. Added to combat. Mirroring Folger.